src/main/resources/public/ts/controllers/controller.ts

```diff
@@ -258,27 +260,28 @@ export const SupportController: Controller = ng.controller('SupportController',
 			}
 		};
 
-		$scope.openTicket = function(id) {
-			if(!$scope.ticket || ($scope.ticket && $scope.ticket.id !== id))
-				$scope.ticket = _.find(model.tickets.all, function(ticket){
+		$scope.openTicket = async function (id) {
+			if (!$scope.ticket || ($scope.ticket && $scope.ticket.id !== id))
+				$scope.ticket = _.find(model.tickets.all, function (ticket) {
 					return ticket.id === id;
 				});
-			if(!$scope.ticket) {
-				if($scope.searchInput){
+			if (!$scope.ticket) {
+				if ($scope.searchInput) {
 					$scope.notFoundSearchInput = true;
 					$scope.searchInput = false;
-				}else{
+				} else {
 					$scope.notFound = true;
 				}
 				$scope.$apply();
 				return;
 			}
-			template.open('main', 'view-ticket');
+			$scope.changeStatusAfterOpenTicket();
```

Comment:
```suggestion
			await $scope.changeStatusAfterOpenTicket();
```
si tu regarde bien, ta fonction "changeStatusAfterOpenTicket" (l.294) est défini en asynchrone, de ce fait, tu dois mettre un await quand tu appels, sinon, ton code va appeler la méthode, puis il va continuer son exécution , sans attendre que la méthode ai fini son traitement !